taichi/backends/opengl/shaders/clear_list.glsl.h

```diff
@@ -0,0 +1,10 @@
+// vim: ft=glsl
```

Comment:
Pls remove. These shouldn't go into this PR. Pls start a separate branch for #711